app/views/posts/index.html.erb

```diff
@@ -6,14 +6,8 @@
       <div class="post">
         <h2><%= post.title %></h2>
         <p><%= post.content %></p>
-        <p>This post has <%= post.likes.count %>
-          <%= pluralize(post.likes.count, "like") %>!
-        </p>
-        <% if liked_by?(post, current_user) %>
-          <%= render "posts/partials/unlike_button", post: @post %>
-        <% else %>
-          <%= render "posts/partials/like_button", locals: {post: @post} %>
-        <% end %>
+        <p>This post has <%= pluralize(post.likes.count, "like") %>! </p>
+        <%= render "posts/partials/like_button", post: post %>
```

Comment:
If you store that view in `posts/_like_button.html.erb` you don't need to specify the path:
```suggestion
        <%= render "like_button", post: post %>
```